static_test_driver/static_test_driver.ino

```diff
@@ -235,40 +222,32 @@ void loop() {
   
   BEGIN_READ
   READ_FLAG(zero_force) {
-    #if CONFIGURATION == MK_2
     Serial.println(F("Zeroing load cell"));
     scale.tare(); // Load Cell, Assuming there is no weight on the scale, reset to 0
-
-    #else
-    if (force_zero_ready) {
-      Serial.println(F("Zeroing force plate"));
-      force_zero_ready = false;
-      force_zero_val = 0;
-      for (int i = 0; i < FORCE_PLATE_NUM_HIST_VALS; i++)
-        force_zero_val += force_hist_vals[i];
-      force_zero_val /= FORCE_PLATE_NUM_HIST_VALS;
-    }
-    else {
-      Serial.println(F("Force zero value not ready"));
-    }
-
-    #endif
   }
-  #if CONFIGURATION == MK_2
-  READ_FLAG(zero_pressure) {
+  
+  //TODO: Update flag names to match the following code 
+  READ_FLAG(zero_pressure_fuel) {
     if (pressure_zero_ready) {
-      Serial.println(F("Zeroing pressure"));
-      pressure_zero_ready = false;
-      pressure_zero_val = 0;
-      for (int i = 0; i < PRESSURE_NUM_HIST_VALS; i++)
-        pressure_zero_val += pressure_hist_vals[i];
-      pressure_zero_val /= PRESSURE_NUM_HIST_VALS;
+      Serial.print(F("Zeroing fuel pressure"));
+      pressure_zero_val[0] = 0;
+      mean(&pressure_hist_vals[0][0], PRESSURE_NUM_HIST_VALS);
```

Comment:
This line doesn't do anything with the value returned by `mean`.  Also `&foo[x][0]` can be rewritten as just `foo[x]`, so these 2 lines should be just
`pressure_zero_val[0] = mean(pressure_hist_vals[0], PRESSURE_NUM_HIST_VALS);`

Same for below.